project/Publish.scala

```diff
@@ -39,13 +42,6 @@ object Publish extends AutoPlugin {
     publishMavenStyle := true,
     pomIncludeRepository := (_ => false))
 
-  private def pekkoPublishTo = Def.setting {
```

Comment:
This is not needed anymore because sbt-sonatype handles this automatically behind the scenes. Hopefully Apache's Nexus behaves in the exact same way as OS sonatype with only a different host, in which case we can use sbt-ci-release.